doc/api/path.md

```diff
@@ -541,6 +541,18 @@ On Windows:
 accepted as path segment separators; however, the `path` methods only add
 backward slashes (`\`).
 
+## path.toLongUNCPath(path)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `path` {string}
+* Returns: {string}
+
+On Windows systems only, returns the equivalent UNC path for the given `path`.
```

Comment:
The description is imprecise. Also if we decide to go with a `resolveExtended` interpretation the description needs to change to include a reference to `resolve`:


```markdown
Resolves the `path` with [`path.resolve()`][].
On Windows systems only, returns an "extended-length" path, which includes the "\\?\" prefix.
For more information see [this MSDN][aa365247]
...
[`path.resolve()`]: #path_path_resolve_paths
[aa365247]: https://msdn.microsoft.com/library/windows/desktop/aa365247(v=vs.85).aspx
```